certified-connectors/OpenText Documentum/readme.md

```diff
@@ -0,0 +1,24 @@
+## OpenText Documentum by One Fox
+Empower OpenText Documentum as the primary repository for all things content related. Together with the OpenText Documentum Power Platform Connector by One Fox you can streamline content processes across a widespread application landscape and limit information silos.
+

```

Comment:
Please update readme.md file as per the template in the below url. 
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
I have changed the PR title and updated the readme.md